service/nodeconfig/v1/resource/node/create.go

```diff
@@ -7,12 +7,19 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/client/k8srestconfig"
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 
 	"github.com/giantswarm/node-operator/service/nodeconfig/v1/key"
 )
 
+const (
+	// UnschedulablePatch is the JSON patch structure being applied to nodes using
+	// a strategic merge patch in order to drain them.
+	UnschedulablePatch = `{"spec":{"unschedulable":true}}`
```

Comment:
When this is used in a single place I don't think it has to be a constant. I think it's easier to read when you don't have to jump around for no reason.

Reply:
I do this for godoc reasons and documentation of the process in general. I do not have to read every single line of all the code everywhere. I can more easily grasp the essentials that way. This bit here is a very important one IMO and having it presented that way is a good thing.